src/turbostat.c

```diff
@@ -1474,35 +1474,22 @@ turbostat_read(void)
 static int
 check_permissions(void)
 {
-#ifdef HAVE_SYS_CAPABILITY_H
-	struct __user_cap_header_struct cap_header_data;
-	cap_user_header_t cap_header = &cap_header_data;
-	struct __user_cap_data_struct cap_data_data;
-	cap_user_data_t cap_data = &cap_data_data;
+
 	int ret = 0;
```

Comment:
`ret` is declared and not used if `HAVE_SYS_CAPABILITY_H` or `CAP_SYS_RAWIO` is undefined.